Runtime/Core/ActionResult.cs

```diff
@@ -61,12 +62,21 @@ public ActionResult(ActionResult.Status status, string notification)
             return res != null && res.status == Status.Success;
         }
 
+        /// <summary>
+        /// Convert the ActionResult to a boolean value, true if successful and false if not.
+        /// </summary>
+        /// <returns>True if action was successful, false otherwise.</returns>
         public bool ToBool()
         {
             return status == Status.Success;
         }
 
-        public static bool FromBool(bool success)
+        /// <summary>
+        /// Convert a boolean value to an ActionResult, Success if true and Failure if false.
+        /// </summary>
+        /// <param name="success">The bool value to convert</param>
+        /// <returns>Success if success is true and Failure if false.</returns>
+        public static ActionResult FromBool(bool success)
         {
```

Comment:
Would propose this be a constructor rather than static method.

I suppose these are already public though. So given that this is just documentation it looks OK to me, but let's deprecate these with the next major version increment.

Reply:
Yeah I agree on that, seems weird regarding the rest of the class to have it as a static method.
I worry that changing the return type here to correct the method is creating an API change which would require a major version bump :/